src/app/service/constants/apiKeys.ts

```diff
@@ -0,0 +1,10 @@
+const apiKeys = Object.freeze({
```

Comment:
Can we rename this file to `TwitterAPIKeys`? Or do you plan on packing this with all API keys in the future?

Reply:
Plan is to pack all api keys here going forward